src/module/actor/SR5Actor.ts

```diff
@@ -387,6 +440,14 @@ export class SR5Actor extends Actor {
         return this.data.data.attributes[attributeName];
     }
 
+    getEquippedMatrixDevice(): SR5Item | undefined {
+        return this.items.filter((item: SR5Item) => item.isDevice())[0];
```

Comment:
.find will give the first match
```suggestion
        return this.items.find((item: SR5Item) => item.isDevice());
```